kerl

```diff
@@ -894,6 +903,23 @@ endif
 
 rehash
 ACTIVATE_CSH
+    if [ -n "$KERL_INSTALL_DOCSH" ]; then
```

Comment:
I think it might be better/easier to make installing the docsh extension a separate command? @sanmiguel what do you think?